src/codecatalyst/auth.ts

```diff
@@ -101,16 +101,18 @@ export class CodeCatalystAuthenticationProvider {
 
         const conn = (await this.auth.listConnections()).find(isBuilderIdConnection)
         const isNewUser = conn === undefined
+        const okItem: vscode.MessageItem = { title: localizedText.ok }
+        const cancelItem: vscode.MessageItem = { title: localizedText.cancel, isCloseAffordance: true }
 
         if (isNewUser || !isValidCodeCatalystConnection(conn)) {
             // TODO: change to `satisfies` on TS 4.9
             telemetry.record({ codecatalyst_connectionFlow: isNewUser ? 'Create' : 'Upgrade' } as ConnectionFlowEvent)
 
             const message = isNewUser
-                ? 'CodeCatalyst requires an AWS Builder ID connection. Creating a connection opens your browser to login. Create one now?'
-                : 'Your AWS Builder ID connection does not have access to CodeCatalyst. Upgrading the connection requires another login. Upgrade now?'
-            const resp = await vscode.window.showInformationMessage(message, localizedText.yes, localizedText.no)
-            if (resp !== localizedText.yes) {
+                ? 'CodeCatalyst requires an AWS Builder ID connection. Creating a connection opens your browser to login.\n\n Create one now?'
+                : 'Your AWS Builder ID connection does not have access to CodeCatalyst. Upgrading the connection requires another login.\n\n Upgrade now?'
+            const resp = await vscode.window.showInformationMessage(message, { modal: true }, okItem, cancelItem)
```

Comment:
When I tested the PR earlier, it seemed to work well as non-modal. Did you find an issue?

Reply:
Notifications can disappear pretty quickly which could confuse users. Modal can be justified here as long as this code path was hit by a user-initiated action.

Could go either way here. Maybe there's a way to keep notifications alive longer?